content/blog/how-to-provision-hpe-greenlake-private-cloud-resources-from-servicenow-service-catalog.md

```diff
@@ -0,0 +1,150 @@
+---
+title: How to provision HPE GreenLake private cloud resources from the
+  ServiceNow Service Catalog
+date: 2022-03-08T06:13:00.000Z
+author: Vinnarasu Ganesan, Thavamaniraja S
+authorimage: /img/Avatar1.svg
+thumbnailimage: /img/thumbnail.jpg
+tags:
+  - hpe-greenlake
+---
+## Introduction
+
+A key strength of HPE GreenLake for private cloud is its self-service
+orchestration and automation, which is done from the HPE GreenLake for
+private cloud dashboard and can be consumed using API and Infrastructure-as-Code (IaC). However,
+some ServiceNow users prefer to provision resources from the ServiceNow
+Service Catalog. For these users, HPE GreenLake for private cloud offers
+a free Morpheus plugin for ServiceNow, which can be installed from the
+ServiceNow Store. Once the plugin is installed and configured, HPE
+GreenLake for private cloud [catalog items](https://developer.hpe.com/blog/curate-and-expose-service-catalog-items-using-hpe-greenlake-for-private-cloud)
+can be presented in the ServiceNow
+Service Catalog for ordering. 
+
+This article walks through the
+process of integrating ServiceNow with HPE GreenLake for private cloud
+and exposing Service Catalog items to ServiceNow.
+
+## Prerequisites
+
+<!--StartFragment-->
+
+The process of integrating HPE GreenLake for private cloud on ServiceNow requires different things depending on your environment. In any case, you will need the Morpheus plugin on ServiceNow, HPE GreenLake for private cloud service user account and Private cloud user in ServiceNow. 
+
+#### Install the Morpheus plugin
+
+To obtain and install the Morpheus plugin, you must have your HI credentials. Using your HI account, simply search for Morpheus and install the Morpheus plugin.
+
+#### Private cloud service user account
+
+Before you begin the integration process, raise a ticket on HPE GreenLake support to create a service user in your HPE GreenLake for private cloud environment. For details, see [Request for sub-tenant service user for ServiceNow integration](https://support.hpe.com/hpesc/public/docDisplay?docId=a00092451en_us&page=request-for-subtenant-service-user-for-servicenow-integration.html).
+
+#### Private cloud user in ServiceNow
+
+Using ServiceNow admin user, create a HPE GreenLake for private cloud user in ServiceNow with the below roles:
+
+* catalog_admin
+* import_transformer
+* itil
+* rest_service
+* xmodamorpheus_ca.integration *(NOTE: You can only assign this role after installing the Morpheus plugin. If you are creating this user before that, you must go back and add this role after installing the plugin.)*
+
+<!--EndFragment-->
+
+## Integrating private cloud on ServiceNow
+
+Configure the Morpheus Catalog in ServiceNow with the properties followed
```

Comment:
I just noticed this one doesn't have any punctuation at the end. In addition, I'm not entirely sure what the sentence means. Does it mean "Configure the Morpheus Catalog in ServiceNow with the following properties:"? If so, then please modify the end to "following properties" and use a colon at the end.

If it means something else, consider rewording, as it's not clear.

Reply:
I reword the sentence as "Configure Morpheus properties in ServiceNow with the following parameters:"